sympy/physics/control/control_plots.py

```diff
@@ -707,6 +709,8 @@ def bode_magnitude_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs)
         The only variable in the transfer function should be
         the variable of the Laplace transform.
 
+    When incorrect frequency units are given as input.
```

Comment:
I think this may cause the sphinx error.  Remove the empty line above and indent this as the text above.